profondo/cnn.py

```diff
@@ -151,5 +251,6 @@ def report_callback(epoch, model, x_val, y_val):
         callbacks=callbacks)
 
     # Load best model and evaluate it on the test data
-    model = load_model(x_test, y_test)
-    report_callback(0, model, x_test, y_test)
+    if exists(MODEL_FILENAME):
```

Comment:
Right, but we really want to evaluate the best model on the test data there so I think it makes sense to assert on the existence of the model.